app/uk/gov/hmrc/apinotificationqueue/model/notificationModels.scala

```diff
@@ -27,6 +27,16 @@ object NotificationStatus extends Enumeration {
   val Pulled = Value("pulled")
 }
 
```

Comment:
I think we could have `case class NotificationId(notificationId: UUID) extends AnyVal` to improve performance